src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1771,31 +1797,31 @@
     short-summary: Name or id of the subscription the new policy set definition can be assigned in.
 examples:
   - name: Create a policy set definition.
-    text: |
-        az policy set-definition create -n readOnlyStorage \\
-            --definitions "[ { \\"policyDefinitionId\\": \\"/subscriptions/mySubId/providers/ \\
+    text: >
+        az policy set-definition create --name readOnlyStorage \\
+            --definitions "[ { \\"policyDefinitionId\\": \\"/subscriptions/MySubscriptionID/providers/ \\
                 Microsoft.Authorization/policyDefinitions/storagePolicy\\", \\"parameters\\": \\
                     { \\"storageSku\\": { \\"value\\": \\"[parameters(\\\\"requiredSku\\\\")]\\" } } }]" \\
             --params "{ \\"requiredSku\\": { \\"type\\": \\"String\\" } }"
   - name: Create a policy set definition with parameters.
-    text: |
-        az policy set-definition create -n readOnlyStorage \\
-            --definitions '[ { \\"policyDefinitionId\\": \\"/subscriptions/mySubId/providers/ \\
+    text: >
+        az policy set-definition create --name readOnlyStorage \\
+            --definitions '[ { \\"policyDefinitionId\\": \\"/subscriptions/MySubscriptionID/providers/ \\
                 Microsoft.Authorization/policyDefinitions/storagePolicy\\" } ]'
   - name: Create a policy set definition in a subscription.
-    text: |
-        az policy set-definition create -n readOnlyStorage \\
+    text: >
+        az policy set-definition create --name readOnlyStorage \\
             --subscription '0b1f6471-1bf0-4dda-aec3-111122223333' \\
             --definitions '[ { \\"policyDefinitionId\\": \\"/subscriptions/ \\
                 0b1f6471-1bf0-4dda-aec3-111122223333/providers/Microsoft.Authorization/ \\
                     policyDefinitions/storagePolicy\\" } ]'
   - name: Create a policy set definition with policy definition groups.
-    text: |
-        az policy set-definition create -n computeRequirements \\
-            --definitions "[ { \\"policyDefinitionId \\": \\"/subscriptions/mySubId/providers/ \\
+    text: >
+        az policy set-definition create --name computeRequirements \\
+            --definitions "[ { \\"policyDefinitionId \\": \\"/subscriptions/MySubscriptionID/providers/ \\
                 Microsoft.Authorization/policyDefinitions/storagePolicy\\", \\"groupNames\\": \\
                     [ \\"CostSaving\\", \\"Organizational\\" ] }, { \\"policyDefinitionId\\": \\
-                        \\"/subscriptions/mySubId/providers/Microsoft.Authorization/ \\
+                        \\"/subscriptions/MySubscriptionID/providers/Microsoft.Authorization/ \\
                             policyDefinitions/tagPolicy\\", \\"groupNames\\": [ \\
```

Comment:
> mySubId

Could we consider replacing placeholders with sample values so that users can understand it's format?